binder/build.gradle

```diff
@@ -91,4 +91,6 @@ tasks.withType(JavaCompile) {
     options.errorprone.check("UnnecessaryAnonymousClass", CheckSeverity.OFF)
 }
 
+tasks.build.dependsOn assembleAndroidTest
```

Comment:
This wouldn't have helped, because android.sh wasn't running it. I don't think we should do this change, or at least not do it here.

Reply:
Right, I'm not fixing CI. Just fixing local build.

Reply:
This isn't standard Android. I think it is better to accept and understand that not everything is built via `build` for Android than changing the dependency list. Even if we thought `build` should build the android test code, we shouldn't make that change here, but instead in `build.gradle.`

Reply:
Reverted.